symengine/tests/eval/test_eval_mpfr.cpp

```diff
@@ -77,5 +78,15 @@ TEST_CASE("precision: eval_mpfr", "[eval_mpfr]")
     REQUIRE(mpfr_cmp_d(a, 2.00000000000001) == -1);
     REQUIRE(mpfr_cmp_d(a, 1.99999999999999) == 1);
```

Comment:
Unrelated to the PR. Same situation as before. `r` is evaluated to `2` and `min` is not simplified. Better test case would have been `min(sqrt(2), 2)`